chromaticqa readme doc (#3283)
bbc/simorgh: docs/Chromaticqa-Guide.md

> # ChromaticQA

Reviewer: This documentation is good, but Chromatic is actually more useful in Psammead. I would move this there and reference it in existing documentation pages.

## Introduction to ChromaticQA
[Chromatic Docs](https://docs.chromaticqa.com/)

> ChromaticQA is a visual regression tool for storybook maintained by Storybook.The purpose of having chromaticQA run on all stories is to capture unwanted visual changes introduced by every change done to the code base.This reduces the overhead on manual regression as it supports screenshot comparisons for chrome,IE and FF.

Reviewer 2: ```suggestion
ChromaticQA is a visual regression tool for storybook maintained by Storybook. The purpose of having chromaticQA run on all stories is to capture unwanted visual changes introduced by every change done to the code base. This reduces the overhead on manual regression as it supports screenshot comparisons for Chrome, IE and FF.
```

## Running Test

> ChromaticQA runs as a part of the build.The builds can be seen here (https://www.chromaticqa.com/builds?appId=5d28eb3fe163f6002046d6fa)

Reviewer 2: ```suggestion
ChromaticQA runs as a part of the build. The builds can be seen here (https://www.chromaticqa.com/builds?appId=5d28eb3fe163f6002046d6fa)
```

> To run chromatic locally export the app code in bash profile(`export CHROMATIC_APP_CODE=<app-code>`) and run `npm run test:chromatic`

Reviewer 2: hmm `npm run test:chromatic`has been removed because of this issue: https://github.com/bbc/simorgh/pull/3374

we can update the doc here to reflect that with 
```suggestion
To run chromatic locally export the app code in bash profile(`export CHROMATIC_APP_CODE=<app-code>`) and run `npx chromatic test run  --build-script-name build:storybook || true`
```

> To run chromatic locally export the app code in bash profile(`export CHROMATIC_APP_CODE=<app-code>`) and run `npx chromatic test run  --build-script-name build:storybook || true`

Reviewer: What is the use case for this? It is expensive to run these tests, I think running on every push should be more than enough.

Reviewer 2: my use case for running it locally :
 
- if I'm developing something to fix a bug regarding our snapshots, then it is quicker to run chromatic tests locally than pushing my changes to a PR. (but I guess this command would only be used heavily in core-pod)

Author: I can only think of the above use case for running it locally

Reviewer: Ok, I guess it's fine to leave it.

- When the tests are run first it creates the baselines screenshots unreviewed for all the browsers enabled in the settings.

> - Each new build creates snapshots of the components which are compared with the baseline screenshots and unchanged ones are automatically accepted and changed ones needs review(accordingly it has to be accepted or denied).If there are new components added they are listed under new components.

Reviewer 2: ```suggestion
- Each new build creates snapshots of the components which are compared with the baseline screenshots and unchanged ones are automatically accepted and changed ones needs review(accordingly it has to be accepted or denied). If there are new components added they are listed under new components.
```

> - Each story has its own baseline that is tracked independent of each branch. When you approve a snapshot you also update the baseline for that story.

Reviewer 2: ```suggestion
- Each story has its own baseline that is tracked independently on each branch. When you approve a snapshot you also update the baseline for that story.
```

# ChromaticQA

## Introduction to ChromaticQA
[Chromatic Docs](https://docs.chromaticqa.com/)

ChromaticQA is a visual regression tool for storybook maintained by Storybook.The purpose of having chromaticQA run on all stories is to capture

> unwanted visual changes introduced by every change done to the code base.This reduces the overhead on manual regression as it supports screenshot

Reviewer: there shouldn't be a line break here, all paragraph should be on line 6

Reviewer: same throughout the doc

Author: yup will make the changes
